lua/telescope/_extensions/file_browser/actions.lua

```diff
@@ -295,12 +295,23 @@ fb_actions.copy_file = function(prompt_bufnr)
       end
     end
     if destination ~= "" then -- vim.fn.input may return "" on cancellation
-      file:copy {
+      local res = file:copy {
         destination = destination,
         recursive = true,
         parents = true,
       }
-      print(string.format("\n%s has been copied!", filename))
+
+      for key, value in pairs(res) do
+        if key.filename == destination then
+          local res_msg
+          if res[key] then
```

Comment:
One thing that comes to mind here (referring to my other post), for instance are copying folders.

IIRC my plenary PR correctly, a folder would return a table of 

```lua
{
  file_a.ext = bool,
  file_b.ext = bool,
  folder = { ... },
}
```

Hence, I think we'd be erroneously printing success, whenever one of the sub-files is copied improperly.

Reply:
Thanks for pointing this out @fdschmidt93 

I think you are onto something, but actually, we are not printing any message for copied folders.

I also noticed something interesting when copying files/folders, we are copying only the files/folders in the current dir and the first sub-dir down, all other deeper levels are recursively copied over, but we don't get a result for those file/directories after the first level down.

I need to take a closer look into this. I'll try to work on this sometime tomorrow.